### PR TITLE
Add tooltip divider lines

### LINKED ISSUE #457: Tool tip needs line grey divider lines
I believe the tool tips get a light grey line to help separate data points. @shixie or @cameroncalder  can provide the spec but assume it’s grey 20. And let me know if I’m wrong here but it seems it needs additional help to separate. 

Screen Shot 2019-12-30 at 12 36 50 PM

### PR DESCRIPTION
**Pull request by sophiiae**
closes #457

### CODE REVIEW

**Comment by sophiiae** [comment]
> You've edited `demo/index.scss` where you should be editing files in the `src/styles` directory.
> 
> So these styles basically only apply in our demo site and not in the actual library

**Comment by sophiiae** [comment]
> You've edited `demo/index.scss` where you should be editing files in the `src/styles` directory.
> 
> So these styles basically only apply in our demo site and not in the actual library

Fixed.

File: packages/core/src/styles/components/_tooltip.scss [hunk 1]
```diff
@@ -79,6 +79,13 @@
 
 				&:not(:last-child){
 					margin-bottom: 2px;
+					@if $carbon--theme == $carbon--theme--g90 {
```
Comment (theiliad) [inline packages/core/src/styles/components/_tooltip.scss]: can we please add a blank line between `margin-bottom: 2px;` and the if-statement?

File: packages/core/src/styles/components/_tooltip.scss [hunk 2]
```diff
@@ -79,6 +79,14 @@
 
 				&:not(:last-child){
 					margin-bottom: 2px;
+
```
Comment (theiliad) [inline packages/core/src/styles/components/_tooltip.scss]:
```suggestion
```

File: packages/core/src/styles/components/_tooltip.scss [hunk 3]
```diff
@@ -79,6 +79,14 @@
 
 				&:not(:last-child){
 					margin-bottom: 2px;
```
Comment (theiliad) [inline packages/core/src/styles/components/_tooltip.scss]:
```suggestion
```

**Pull request merged**